apps/admin/frontend/src/app.test.tsx

```diff
@@ -444,7 +443,7 @@ test('tabulating CVRs', async () => {
   advanceTimers(2);
   await screen.findByText('Save Batch Results');
   await screen.findByText(
-    'votingworks-live-batch-results_choctaw-county_mock-general-election-choctaw-2020_2020-11-03_22-22-00.csv'
+    'votingworks-live-batch-results_choctaw-county_mock-general-election-choctaw-2020_2020-11-03_22-22-02.csv'
```

Comment:
Since we're using Jest's fake timers to control `Date`, advancing the timers now affects the current date.